README.md

```diff
@@ -11,16 +16,16 @@ $ npm install crocks -S
 This will pull down `crocks` into your project's `node_modules` folder and can be accessed by adding something like the following in the file that needs it:
 
 ```javascript
-// node require syntax
+// node require syntax - "es5"
 const crocks = require('crocks')
 
-// Javascript modules (if you are transpiling or using rollup.js)
+// Javascript modules (if you are transpiling or using rollup.js) - "es6"
```

Comment:
they are more commonly referred in this aspect.  the quotes entail your point of view on the terms